### ISSUE TITLE
Improve documentation regarding precise fan control (and other suggestions)

### ISSUE THREAD

**Issue by nikp123** [marhop/fanslow-probook430g2#1]
Big thanks for this repo. However, I'd like for information to be more accessible regarding how to do precise fan control.

For anyone curious, follow along. To set manual fan speed, you do the following:
  1. Install probook_ec script
  2. Then the speed that you want to set the fan on should be: ``# probook_ec := 0x2f 0xff minus your_speed (in hex)``

Also one other interesting idea is to contribute this to https://github.com/hirschmann/nbfc instead, as it holds a large database of existing laptops that support manual fan control.

Anyways, you don't have to do any of these, but I'd like for this to be more streamlined and accessible.

Again, big thanks for this.

**Comment by marhop** [comment]
That sounds like some good ideas, I'm looking forward to seeing your progress!

> Anyways, you don't have to do any of these, [...]

Good. :-) I fear I wouldn't be of much help anyway - I haven't written the central scripts (fanslow and probook_ec), I barely understand them at all. ;-) I only added the README, Makefile and the systemd service unit ...

**Comment by nikp123** [comment]
Yeah, understandable. But you did me a favor, I would've never tried the script from that ubuntu post. Anyways once I get some free time on my hands I'll be sure to look into this. Cheers
